sormas-backend/src/main/java/de/symeda/sormas/backend/deletionconfiguration/CoreEntityType.java

```diff
@@ -0,0 +1,30 @@
+package de.symeda.sormas.backend.deletionconfiguration;
+
+import de.symeda.sormas.api.deletionconfiguration.DeletionReference;
+
+public enum CoreEntityType {
+
+	//ENTITY LIST
+	CASE(DeletionReference.CREATION, 3650),
```

Comment:
Don't define default deletion periods & references.
Automatic deletion will have to be configured by someone on database level in order to work.

Reply:
updated